packages/svelte/README.md

```diff
@@ -7,124 +7,333 @@
 **[Storybook demo sources](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core/demo/data)**
 
 ## Maintenance & support
+
 These Svelte wrappers have been developed by Eric Liu.
 
-Please direct all questions regarding support, bug fixes, and feature requests to [@metonym](https://github.com/metonym).
+Please direct all questions regarding support, bug fixes, and feature requests
+to [@metonym](https://github.com/metonym).
 
 ## Getting started
+
 Run the following command using [npm](https://www.npmjs.com/):
 
 ```bash
-npm install -S @carbon/charts @carbon/charts-svelte d3@5.x
+npm install -D @carbon/charts-svelte d3@5.x
 ```
 
 If you prefer [Yarn](https://yarnpkg.com/en/), use the following command
 instead:
 
 ```bash
-yarn add @carbon/charts @carbon/charts-svelte d3@5.x
+yarn add -D @carbon/charts-svelte d3@5.x
```

Comment:
are we sure? dev-dep?

Reply:
Yes. This is standard for installing Svelte components, particularly for Sapper.